src/Statement/MysqlStatementBuilder.php

```diff
@@ -117,6 +117,14 @@ protected function buildType(array $options): string
                 return 'JSON';
             case 'enum':
                 return sprintf("ENUM('%s')", implode("','", $options['values']));
+            case 'tinytext':
```

Comment:
Having a separate datatype for each text size in MySQL breaks some of the abstraction currently offered (e.g. `string` becoming `varchar`).

Could we approach this by keeping one `text` type and then using the appropriate MySQL type based on an additional `length` option? If no length is provided, the default would be used.

Usage would look like this in order to use `MEDIUMTEXT`:

```php
return \Exo\Migration::create('events')
    ->addColumn('id', ['type' => 'uuid', 'primary' => true])
    ->addColumn('description', ['type' => 'text', 'length' => 100000]);
```

Reply:
Great feedback! Love the mindset of trying to bring some abstraction.

I have implemented changes that check the length of the text and assigns the correct data type.